lib/shoulda/matchers/action_controller/permit_matcher.rb

```diff
@@ -305,18 +308,10 @@ def actual_permitted_parameter_names
           end
         end
 
-        def permit_called?
-          actual_permitted_parameter_names.any?
-        end
-
         def unpermitted_parameter_names
           expected_permitted_parameter_names - actual_permitted_parameter_names
         end
 
-        def verified_permitted_parameter_names
```

Comment:
Good catch, thanks! As far as I know this is always how the `permit` matcher has worked, so I hesitate to change the functionality now. Otherwise, that would make sense.